crates/server/src/datastore/memory.rs

```diff
@@ -210,7 +211,7 @@ impl DataStore for MemoryDataStore {
     async fn store_package_record(
         &self,
         log_id: &LogId,
-        _name: &str,
+        _package_id: &PackageId,
```

Comment:
why/how is this unused?

Reply:
The in-memory data store doesn't keep the names around, but the postgres one does (really only so one can look at the actual SQL table and see the package ids are; they aren't read out by the server).

Until we have a query API, it really helps the operator to figure out what logs are present.